lighthouse-core/config/index.js

```diff
@@ -241,19 +305,53 @@ class Config {
       configJSON = defaultConfig;
     }
 
-    this._audits = configJSON.audits ? expandAudits(
-        filterAudits(configJSON.audits, auditWhitelist)
+    this._configJSON = this._initRequirePaths(configJSON);
```

Comment:
need to not modify `configJSON` in place as `_initRequirePaths` does, as the config is a live object and e.g. used repeatedly by the extension (I wonder if we should just have `Config` expect a string `configJSON` the it `JSON.parse`s so that it's always a new object and we can avoid being careful about this)